feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/chatlist/MetisSearchPagingSource.kt

```diff
@@ -30,8 +31,12 @@ class MetisSearchPagingSource(
         )
             .map(
                 mapSuccess = { loadedPosts ->
+                    // Currently the server returns duplicate posts when searching for posts. This
+                    // caused a crash in the UI.
+                    // Here is the related issue on Github: https://github.com/ls1intum/Artemis/issues/9709
+                    val filteredPosts = loadedPosts.distinctBy { it.id }
```

Comment:
Please add an issue on GitHub that links to the other issue. So that we will remove this workaround as soon as the problem in the original app has been fixed. Add a TODO with the link to the issue in the code here.